src/sentry/static/sentry/app/views/projectInstall/overview.jsx

```diff
@@ -26,7 +27,8 @@ class ProjectInstallOverview extends React.Component {
 
   redirectToDocs = platform => {
     let {orgId, projectId} = this.props.params;
-    let rootUrl = `/${orgId}/${projectId}/settings/install`;
+    let prefix = recreateRoute('', {...this.props, stepBack: -3});
```

Comment:
i was somewhat worried about this change because there are multiple routes that this code runs on, but this seems correct as far as i can test

Reply:
It's used in 3 places right? "New Project", "Getting Started", and "Project Settings -> Error Tracking"?

Reply:
Ah I didn't test this flow: https://percy.io/getsentry/sentry/builds/621717/view/31544486/1280?mode=diff